ArcGISRuntimeSDKQt_QMLSamples/Routing/NavigateARouteWithRerouting/NavigateARouteWithRerouting.qml

```diff
@@ -0,0 +1,340 @@
+// [WriteFile Name=NavigateRoute, Category=Routing]
+// [Legal]
+// Copyright 2022 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+import QtQuick 2.6
+import QtQuick.Controls 2.2
+import Esri.ArcGISRuntime 100.15
+import QtQuick.Layouts 1.11
+import QtPositioning 5.2
+import Esri.ArcGISExtras 1.1
+import Esri.samples 1.0
+
+Rectangle {
+    id: rootRectangle
+    clip: true
+    width: 800
+    height: 600
+
+    readonly property url dataPath: System.userHomePath + "/ArcGIS/Runtime/Data/tpkx/"
```

Comment:
same comment as c++ - consider putting in different folder since it is a geodatabase, not tpkx